src/ThemedWrappers.h

```diff
@@ -0,0 +1,154 @@
+/*!********************************************************************
```

Comment:
You also took this header file out of `src/widgets`

Actually I don't mind this file being in that folder, because it is more general-purpose and reusable.

I only wanted `RealtimeEffectPanel.h` to move out of it.

But this file can stay in `src` too